tests/helper_test.py

```diff
@@ -129,6 +129,18 @@ def test_unicode(self):
         expected = ['echo "foo"', 'sleep 1']
         self._test(base, expected)
 
+    @unittest.expectedFailure
+    def test_doublequoted_semicolons(self):
+        base = 'python -c "import alot; print(alot)"; echo "done"'
+        expected = ['python -c "import alot; print(alot)"', ' echo "done"']
+        self._test(base, expected)
+
+    @unittest.expectedFailure
+    def test_singlequoted_semicolons(self):
+        base = "python -c 'import alot; print(alot)'; echo 'done'"
```

Comment:
This is a string to test the splitting function. It is not to be executed. I just wipped something together which looks like valid shell code in order to use my intuition to find the desired result.